core/src/main/java/bisq/core/trade/protocol/SellerAsTakerProtocol.java

```diff
@@ -59,6 +59,7 @@ public SellerAsTakerProtocol(SellerAsTakerTrade trade) {
         super(trade);
         Offer offer = checkNotNull(trade.getOffer());
         processModel.getTradingPeer().setPubKeyRing(offer.getPubKeyRing());
+        processModel.getTradeManager().requestPersistence();
```

Comment:
I get a NPE here, processModel.tradeManager is null.

Reply:
Ah in the constructor it must not be called as trademanager is not set yet. I will remove that.